### ISSUE TITLE
jaws cannot read <div> tag data

### ISSUE THREAD

**Issue by siddeshks** [FreedomScientific/VFO-standards-support#182]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to http://www.google.com
> 2. Shift-Tab back to the About link

### Expected result

> What did you expect to happen?

### Actual result

> What actually did happen?

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...
